apps/prairielearn/src/migrations/20240307213407_course_requests__referral_source__add.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE course_requests
+ADD COLUMN referral_source TEXT;
```

Comment:
We could make this `NOT NULL` and update existing rows with `"None"`; I figured allowing NULLs to represent "no referral source" was fine though.